docs/modules/provisioning-app/pages/configuration.adoc

```diff
@@ -57,6 +57,13 @@ The configuration for the permission sets are configured:
 . Bitbucket Project is provisioned with tight read & write roles
 . Openshift Project roles linked to the passed groups (`READONLY` - `view`, `ADMINGROUP` - `admin`, `USERS` - `edit`)
 
+Furthermore if you need to define default permission for openshift (e.g. to setup membership permission for cluster admins) you can add this to your application properties:
+```
+jenkinspipeline.create-project.default-project-groups=ADMINGROUP=<MY_CLUSTER_ADMIN_GROUP_NAME>
```

Comment:
How would this be set in the `ConfigMap`? They are defined in https://github.com/opendevstack/ods-core/blob/3.x/ods-provisioning-app/ocp-config/cm.yml. I wonder where to set it so that no drift shows up during verification.

Reply:
@michaelsauter this is the PR for ods-core/3.x: https://github.com/opendevstack/ods-core/pull/889... what do you mean with "no drift shows up during verification"?

Reply:
Ah okay got it. Looks fine!